database/index.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+module.exports = () => {
+  return new Promise((resolve) => {
+    let url = "";
+
+    switch (process.env.ENVIRONMENT) {
+      case "development":
+        url = process.env.DATABASE_URL_DEVELOPMENT;
+        break;
+
+      case "staging":
+        url = process.env.DATABASE_URL_STAGING;
+        break;
+
+      case "test":
+        url = process.env.DATABASE_URL_TEST;
+        break;
+
+      case "production":
+        url = process.env.DATABASE_URL_PRODUCTION;
+        break;
+
+      default:
```

Comment:
This should be moved to config folder `config/databaseConfig.js`

Reply:
what should be moved?

Reply:
Okay let me do that

Reply:
The database configuration file

Reply:
Line 4 could you please indicate why you are returning promises

Reply:
@abidex4yemi I am using a promise because, sometimes especially in cases where you need to spin up a database instance like a seperate connection e.g in cron jobs, some operations require that the database connection be made before those operations are carried out. say for instance you want to delete an old events(I know we are not doing this, its just an example), you will need to have established a database connection before deleting, other wise there will be some error